app/helpers/cookies_helper.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module CookiesHelper
+  def hide_cookie_banner?
+    cookies['consented-to-cookies'].present? || current_page?(cookie_preferences_path)
```

Comment:
I think the banner is supposed to be displayed everywhere if you haven't confirmed (not 100% sure though)